app/src/main/java/ch/epfl/sweng/swenggolf/profile/ProfileActivity.java

```diff
@@ -60,6 +71,29 @@ private void showEditButton() {
         if (user.getUserId().equals(Config.getUser().getUserId())) {
             ImageButton button = findViewById(R.id.edit);
             button.setVisibility(View.VISIBLE);
+        } else {
```

Comment:
That is a good idea to reuse the check of the edit button, but I think that the method should now have another name since it does more than that...

Reply:
You're right. I changed to a more meaningful name.